deck-common/src/commonMain/kotlin/com/deck/common/content/node/Node.kt

```diff
@@ -94,6 +95,12 @@ public sealed class Node(
             data = NodeData(children = listOf(Paragraph.Text(leaves = listOf(Paragraph.Text.Leaf(text)))))
         )
     }
+
+    public class Mention(public val mentionData: RawMentionData) : Node(
```

Comment:
It would be best not to pass the whole mention data object here, but since you said you need help with this part, I will try to find a new solution.

Reply:
I'll try to work on this issue today.